pkg/experiment/symbolizer/types.go

```diff
@@ -0,0 +1,32 @@
+package symbolizer
+
+import (
+	googlev1 "github.com/grafana/pyroscope/api/gen/proto/go/google/v1"
+	"github.com/grafana/pyroscope/lidia"
+
+	pprof "github.com/google/pprof/profile"
+)
+
+type locToSymbolize struct {
```

Comment:
I think we can replace locToSymbolize with just `*googlev1.Location` because `loc.Id == idx +1`